tests/FakeItEasy.IntegrationTests/ContainerTests.cs

```diff
@@ -1,34 +1,13 @@
 namespace FakeItEasy.IntegrationTests
 {
     using System;
-    using System.Collections.Generic;
-    using Creation;
-    using FakeItEasy.Core;
+    using FluentAssertions;
     using NUnit.Framework;
 
     [TestFixture]
     public class ContainerTests
     {
         [Test]
-        public void Creating_fakes_will_use_the_container_of_the_current_scope()
-        {
-            var container = new DictionaryContainer
-            {
-                RegisteredTypes = new Dictionary<Type, object>
-                {
-                    { typeof(string), "fake string" },
-                    { typeof(int), 100 }
-                }
-            };
-
-            using (Fake.CreateScope(container))
-            {
-                Assert.That(A.Dummy<string>(), Is.EqualTo("fake string"));
-                Assert.That(A.Dummy<int>(), Is.EqualTo(100));
-            }
-        }
-
-        [Test]
         public void Should_load_configurations_from_executing_assembly_using_default_container()

```

Comment:
Good point. Really, creating a Dummy from a Dummy Factory is so well covered in the Specs we could delete this class with no ill effects, I think.